testnet/stacks-node/src/tests/signer/mod.rs

```diff
@@ -630,7 +626,7 @@ impl<S: Signer<T> + Send + 'static, T: SignerEventTrait + 'static> SignerTest<Sp
                     }
                 })
                 .collect::<Vec<_>>();
-            Ok(block_rejections.len() == expected_signers.len())
+            Ok(block_rejections.len() >= expected_signers.len())
```

Comment:
Unfortunately could not filter on a specific block hash so cannot properly filter the block responses hence the change to >= as we could have multiple rejections from earlier.

Reply:
I think something along the lines of what I do in this commit is better (https://github.com/stacks-network/stacks-core/pull/5238/commits/6a3746c9f6ca4edec8e9ce66efdbf074e87f5615) -- otherwise, you end up letting the same signer's rejections get counted multiple times because of the way the chunk events work.